analyses/cell-type-ewings/scripts/aucell-ews-signatures/01-aucell.R

```diff
@@ -96,8 +102,19 @@ fs::dir_create(output_dir)
 sce <- readr::read_rds(opt$sce_file)
 
 # remove genes that are not detected from SCE object 
-genes_to_remove <- rowData(sce)$detected > 0 
-filtered_sce <- sce[genes_to_remove , ]
+if(!opt$is_merged){
+  genes_to_remove <- rowData(sce)$detected > 0 
```

Comment:
switching name since logic
```suggestion
  genes_to_keep <- rowData(sce)$detected > 0 
```